src/ProjectService.Business/Commands/Image/CreateImageCommand.cs

```diff
@@ -0,0 +1,135 @@
+﻿using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Constants;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.FluentValidationExtensions;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Enums;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.Models.Broker.Requests.Image;
+using LT.DigitalOffice.Models.Broker.Responses.Image;
+using LT.DigitalOffice.ProjectService.Business.Commands.Task.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using LT.DigitalOffice.ProjectService.Validation.Interfaces;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+namespace LT.DigitalOffice.ProjectService.Business.Commands.Task
+{
+    public class CreateImageCommand : ICreateImageCommand
+    {
+        private readonly IImageRepository _repository;
+        private readonly IRequestClient<ICreateImagesRequest> _rcImages;
+        private readonly ILogger<CreateImageCommand> _logger;
+        private readonly IAccessValidator _accessValidator;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly IDbProjectImageMapper _dbProjectImageMapper;
+        private readonly ICreateImageValidator _validator;
+
+        private List<Guid> CreateImage(List<ImageContent> context, Guid userId, List<string> errors)
+        {
+            List<CreateImageData> images = context
+                .Select(x => new CreateImageData(x.Name, x.Content, x.Extension, userId))
+                .ToList();
+
+            string errorMessage = "Can not create images. Please try again later.";
+            const string logMessage = "Errors while creating images for user id {userId}. Errors: {Errors}";
+
+            try
+            {
+                Response<IOperationResult<ICreateImagesResponse>> brokerResponse = _rcImages.GetResponse<IOperationResult<ICreateImagesResponse>>(
+                   ICreateImagesRequest.CreateObj(images, ImageSource.Project)).Result;
```

Comment:
```suggestion
IOperationResult<ICreateImagesResponse>> brokerResponse = _rcImages.GetResponse<IOperationResult<ICreateImagesResponse>>(
                   ICreateImagesRequest.CreateObj(images, ImageSource.Project)).Result.Message
```

to not write brokerResponse.Message every time

Reply:
done